### DISCUSSION TITLE
Photos not visible

### DISCUSSION THREAD

**Discussion by wtrdk** [photoview/photoview#72] [Q&A and Troubleshooting] [answered]
Hi,

Let me start by saying that I'm very enthousiastic about this project. This is exactly what I was looking for! Great work!

I installed it with docker-compose, but after scanning the files the photos aren't showing. I don't see any errors in the logs:

`2020/08/24 12:46:24 Processing photo: /photos/2020-03/2020-03-03_16.59.03.jpg
2020/08/24 12:46:24 Broadcasting notification: Processing media for album '2020-03'
2020/08/24 12:46:24 Processing video: /photos/2020-03/2020-03-04 18.28.32.mov
2020/08/24 12:46:29 Broadcasting notification: Processing media for album '2020-01'
2020/08/24 12:46:29 Processing photo: /photos/2020-01/2020-01-17 12.31.00.jpg
2020/08/24 12:46:29 Broadcasting notification: Processing media for album '2020-01'
2020/08/24 12:46:29 Processing photo: /photos/2020-01/2020-01-17 14.05.45.jpg
2020/08/24 12:46:30 Broadcasting notification: Processing media for album '2020-01'
2020/08/24 12:46:30 Processing photo: /photos/2020-01/2020-01-17 14.50.10.jpg
2020/08/24 12:46:31 Broadcasting notification: Processing media for album '2020-01'
2020/08/24 12:46:31 Processing photo: /photos/2020-01/2020-01-17 17.54.17.jpg
2020/08/24 12:46:32 Broadcasting notification: Processing media for album '2020-01'
2020/08/24 12:46:32 Processing photo: /photos/2020-01/2020-01-17 17.54.33.jpg
2020/08/24 12:46:32 Broadcasting notification: Processing media for album '2020-01'`

Do you know what's going on? Is there something I need to do with permissions in the photos folder?

<img width="1350" alt="Schermafbeelding 2020-08-24 om 14 39 22" src="https://user-images.githubusercontent.com/4569244/91045880-ce6b5b80-e617-11ea-8633-6ee3940446c2.png">

**Comment by viktorstrate** [comment, marked as answer]
Hi @wtrdk,

Thank you for finding an interest in the project!
I think this problem occures because the server (API) thinks it is hosted on another url that it actually is.

Have you properly configured the `PUBLIC_ENDPOINT` variable in the docker-compose.yml file?
It should be set to the same url as the one shown in your address bar in your browser.

Otherwise, you can debug the url of the images from the Network tab in the developer tools from your browser (Press F12 and click Network). See the image below.

<img width="1381" alt="Screenshot 2020-08-24 at 20 40 43" src="https://user-images.githubusercontent.com/4233458/91083219-333fa980-e64a-11ea-90b6-f2b12899bdf0.png">

**Comment by wtrdk** [comment]
I had the IP set to localhost, but of course that refers to the docker instance, not to the host IP. I changed it to the IP of my server and it works like a charm. Thanks for your response!

**Comment by viktorstrate** [comment]
Awesome! I'll close this issue then.
Feel free to open new issues, if you have a suggestion or find a bug.
